IO-pc-local/src/app/polls/saved-polls.service.ts

```diff
@@ -39,33 +32,23 @@ export class SavedPollsService {
   }
 
   addPoll(newPoll: Poll): void {
-    this.currentPolls.push({
-      ...newPoll,
-      id: this.firstFreeId,
-    })
-    this.firstFreeId++
+    this.httpClient.post<any>(`${API_URL}/poll/${newPoll.name}/save`,newPoll)
```

Comment:
Nie odpala. Trzeba dodać na końcu
```.subscribe(response => {})```
Wtedy odpali. Ale możliwe że to wgl jest zły sposób, nie znam się na tyle.
[https://stackoverflow.com/questions/51787101/angular-http-post-not-firing](https://stackoverflow.com/questions/51787101/angular-http-post-not-firing)

Reply:
Ponadto, front wysyła coś takiego:
![image](https://github.com/Nepommuck/Software-Engineering-Frontend/assets/29920157/0470677b-c4a1-47a2-a53a-6e555e4a0bf2)
A serwer oczekuje czegoś takiego:
![image](https://github.com/Nepommuck/Software-Engineering-Frontend/assets/29920157/049dce46-ae9c-4621-91b7-349f4376d4c1)

Dlaczego model jest rozjechany - nie wiem. Może są jakieś metody konwertujące to gdzieś na froncie?